### ISSUE TITLE
Batch Normalization and Bias

### ISSUE THREAD

**Issue by puhach** [udacity/deep-learning-v2-pytorch#200]
It has come to my notice that bias is always set to False in conv/deconv functions in Deep Convolutional GANs and Cycle GAN projects:

```
def conv(in_channels, out_channels, kernel_size, stride=2, padding=1, batch_norm=True):
    """Creates a convolutional layer, with optional batch normalization.
    """
    layers = []
    conv_layer = nn.Conv2d(in_channels=in_channels, out_channels=out_channels, 
                           kernel_size=kernel_size, stride=stride, padding=padding, bias=False)
    
    layers.append(conv_layer)

    if batch_norm:
        layers.append(nn.BatchNorm2d(out_channels))
    return nn.Sequential(*layers)
```

As far as I understood, **bias** should be set to _False_ only when we apply batch normalization, so it would be more reasonable to set `bias=not batch_norm`.

**Comment by abhiojha8** [comment]
Though we are a little late to respond, we encourage such _user-specific_ issues to be raised on the Knowledge hub, so that our experienced mentors can chime-in and new students can also learn from the discussion thread. 
Thank you
